osi_object.proto

```diff
@@ -745,66 +748,133 @@ message MovingObject
             //
+            // This is one of the OSI types which map from "car" in both OpenScenario 1.x and 2.x.
+            //
             TYPE_LUXURY_CAR = 5;
 
             // Vehicle is a delivery van.
             //
             // Definition: A delivery van.
             //
+            // This corresponds to "van" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "truck" in OpenScenario 2.x.
+            //
             TYPE_DELIVERY_VAN = 6;
 
-            // Vehicle is a heavy truck.
+            // Vehicle is a (heavy) truck.
+            //
+            // Definition: A rigid-body truck that cannot pull a semi-trailer.
+            //
+            // This corresponds to "truck" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "truck" in OpenScenario 2.x.
             //
             TYPE_HEAVY_TRUCK = 7;
 
-            // Vehicle is a truck with semitrailer.
+            // Vehicle is a tractor unit capable of pulling a semi-trailer.
+            //
+            // Definition: The cab for an articulated vehicle.
+            //
+            // This has no equivalent in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "truck" in OpenScenario 2.x.
+            //
+            TYPE_SEMITRACTOR = 16;
+
+            // This vehicle is a trailer unit for an articulated vehicle.
+            //
+            // Definition: A trailer that can be pulled by a semi-tractor.
+            // There is no precise definition of this vs a typical trailer, although
+            // typically it is used for trailers with a non-trivial bounding box
+            // overlap with the tractor unit.
+            //
+            // This corresponds to "semitrailer" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "trailer" in OpenScenario 2.x.
             //
             TYPE_SEMITRAILER = 8;
 
             // Vehicle is a trailer (possibly attached to another vehicle).
             //
+            // Definition: A trailer that isn't pulled by a semi-tractor.
+            //
+            // This corresponds to "trailer" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "trailer" in OpenScenario 2.x.
+            //
             TYPE_TRAILER = 9;
 
             // Vehicle is a motorbike or moped.
             //
+            // This corresponds to "motorbike" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "VRU_vehicle" in OpenScenario 2.x.
+            //
             TYPE_MOTORBIKE = 10;
 
             // Vehicle is a bicycle (without motor and specific lights).
             //
+            // This corresponds to "bicycle" in OpenScenario 1.x.
+            //
+            // This is one of the OSI types which map from "bus" in OpenScenario 2.x.
```

Comment:
Probably copy paste typo. I think it should map to VRU_vehicle from OpenSCENARIO 2.0
```suggestion
            // This is one of the OSI types which map from "VRU_vehicle" in OpenScenario 2.x.
```

Reply:
Good spot, fixed!